engine-tests/build.gradle

```diff
@@ -49,5 +49,6 @@ dependencies {
     compile group: 'junit', name: 'junit', version: '4.12'
     compile group: 'org.mockito', name: 'mockito-all', version: '1.10.19'
     compile group: 'org.jboss.shrinkwrap', name: 'shrinkwrap-depchain-java7', version: '1.1.3'
+    compile group: 'org.evosuite', name:'evosuite-standalone-runtime', version: '1.0.4'
```

Comment:
I'm not sure if "evosuite-standalone-runtime" is necessary. In this test, only `verifyException()` method is used from the library. Maybe we can just remove these tests with `verifyException()` if we don't want use evosuite library